packages/apps-config/src/api/spec/unique.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright 2017-2021 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { OverrideBundleDefinition } from '@polkadot/types/types';
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+const definitions: OverrideBundleDefinition = {
+  types: [
+    {
+        "AccessMode": {
```

Comment:
This should auto-fix if you run `yarn lint` locally.

Reply:
Thanks, Jaco! Running it now...